openupgrade_scripts/scripts/mail/15.0.1.5/pre-migration.py

```diff
@@ -0,0 +1,33 @@
+
+# Copyright 2020 Odoo Community Association (OCA)
+# Copyright 2020 Opener B.V. <stefan@opener.am>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import logging
+
+from openupgradelib import openupgrade
+
+_logger = logging.getLogger(__name__)
+
+
+@openupgrade.migrate(use_env=False)
+def migrate(cr, version):
+    # Cleanup potential duplicates
+    openupgrade.logged_query(
+        cr,
```

Comment:
```suggestion
@openupgrade.migrate()
def migrate(env, version):
    # Cleanup potential duplicates
    openupgrade.logged_query(
        env.cr,
```
This is @pedrobaeza preference and he has long convinced me to do this way as default.

Reply:
You never know when you are going to need the environment, so better to have it already populated for not having a great diff later.